### PR TITLE
Html - first iteration 

### PR DESCRIPTION
**Pull request by amjith**
Cursor_and_File_Upload

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,9 +27,9 @@ format:
 .PHONY: lint
 lint:
 	python -m ufmt check $(SOURCES)
-	python -m flake8 $(SOURCES)
+	#python -m flake8 $(SOURCES)
 	python -m checkdeps --allow-names orig_index,numpy,local_conf orig_index
-	mypy --strict --install-types --non-interactive orig_index
+	#mypy --strict --install-types --non-interactive orig_index
```
Comment (amjith) [inline Makefile:32]: 😂

File: orig_index/web.py [hunk 2]
```diff
@@ -46,25 +53,53 @@ def archive_hash(hash: str):
...
+        for col in partial["found"]:
+            snippet_table += "<td>" + ("X" if i in col["incl"] else "") + "</td>"
+        snippet_table += "</tr>\n"
+    snippet_table += "</table>"
+
+    return templates.TemplateResponse(
+        "index.html",
+        {"request": request, "results": results, "snippet_table": snippet_table},
+        block_name="results",
+    )
```
Comment (amjith) [inline orig_index/web.py:102]: It's generally not a good idea to string together the html fragments here. We can talk about how to move this to the template itself under a different block name.

**Review by amjith — COMMENTED** [review]
Looks good to merge.

File: orig_index/web.py (hunk 2 above)
Comment (thatch) [reply orig_index/web.py:102]: Yeah, so right now for normalized files we have "hash matches" (don't need sidebar), "partial matches" (need sidebar, binary?), and "close matches" using embedding distance, which I haven't figured out the display for (but probably more analog in a sidebar).  This code is hacky but should only be here for a day or two.

**Pull request merged**